### DISCUSSION TITLE
Extract 2D Arbitrary Plane as Numpy array

### DISCUSSION THREAD

**Discussion by mmcint** [pyvista/pyvista#5129] [Q&A]
I'm working with a 3D array of seismic data. I've created a 2D arbitrary line through the 3D array using `add_volume_clip_plane()`
Seen here: 
![screenshot](https://github.com/pyvista/pyvista/assets/135659585/391687cb-f470-4201-be86-b4faccd5eee9)
The red box represents the 2D plane. I am trying to understand if it is possible to convert this 2D plane into a 2D numpy array and export, rather than just a screen shot of the full visualization. I don't see this functionality discussed in the documentation.

**Comment by banesullivan** [comment]
Hey! I wrote up a post on this a long while back:  https://github.com/pyvista/pyvista-support/issues/89#issuecomment-574394910 (and see comments lower in that thread)

See also https://github.com/pyvista/pyvista-support/issues/70

**Reply by mmcint** [reply]
Awesome. Thanks Bane! I'll check it out.

**Reply by banesullivan** [reply]
If you make a nice implementation of this, it'd be great to share and/or contribute to PyVista. I think there is general use for the workflow of "slice volumetric data and plot or process that slice in another library"
